DataPlotly/gui/plot_settings_widget.py

```diff
@@ -1374,7 +1373,7 @@ def create_plot_factory(self) -> PlotFactory:
         plot_factory = PlotFactory(settings, visible_region=visible_region)
 
         # unique name for each plot trace (name is idx_plot, e.g. 1_scatter)
-        self.pid = ('{}_{}'.format(str(self.idx), settings.plot_type))
+        self.pid = (f'{str(self.idx)}_{settings.plot_type}')
```

Comment:
code clean test is failing here: https://github.com/ghtmtt/DataPlotly/actions/runs/4977948527/jobs/8907624514?pr=323